stdlib/Sockets/src/Sockets.jl

```diff
@@ -614,6 +626,14 @@ To listen on all interfaces pass `IPv4(0)` or `IPv6(0)` as appropriate.
 `backlog` determines how many connections can be pending (not having
 called [`accept`](@ref)) before the server will begin to
 reject them. The default value of `backlog` is 511.
+
+# Examples
+```jldoctest
+julia> server = listen(9000)
+Sockets.TCPServer(RawFD(24) active)
+
+julia> close(server);
```

Comment:
Maybe this doesn't need to be a doctest? Or we could pick another port, just so we can avoid having to have `close` in here too.